### PR TITLE
Create schema for SC-237A form

### PR DESCRIPTION
**Pull request by rhofvendahl**
This PR creates a schema for an  SC-237A form.

References:
https://www.ams.usda.gov/resources/sc237a
https://www.ams.usda.gov/sites/default/files/media/SC237A.pdf

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

**Review by BenjaminMoe — CHANGES_REQUESTED** [review]
`SC237AForm` should use the following format for `type`. 

```
    type: array
    readOnly: true
    const:
      - SC237AForm
    default:
      - SC237AForm
    items:
      type: string
      enum:
        - SC237AForm
```

Also add `required` after `additionalProperties: false`
```
required:
  - type
```

File: docs/openapi/components/schemas/common/SC237AForm.yml [hunk 1]
```diff
@@ -0,0 +1,189 @@
+$linkedData:
+  term: SC237AForm
+  '@id': https://w3id.org/traceability#SC237AForm
+title: SC-237A Form
+description: Information regarding an application for audit services.
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/SC237AForm.yml:5]: Can we link to a hosted version of this form from here? We're doing similar for other forms.

Comment (rhofvendahl) [reply docs/openapi/components/schemas/common/SC237AForm.yml:5]: @mkhraisha 

Do you mean the pdf? As in https://www.ams.usda.gov/sites/default/files/media/SC237A.pdf ?

I'm concerned that a link like that might break, but a link to the USDA page for that form (https://www.ams.usda.gov/resources/sc237a) should work, as it references the form and is more likely to be updated.

I'm open to doing that, but would it work instead to just have a more detailed description (without breakable links)? I've been meaning to do that anyway.

**Review by BenjaminMoe — APPROVED** [review]

File: docs/openapi/components/schemas/common/SC237AForm.yml [hunk 2]
```diff
@@ -0,0 +1,193 @@
+$linkedData:
+  term: SC237AForm
+  '@id': https://w3id.org/traceability#SC237AForm
+title: SC-237A Form
```
Comment (nissimsan) [inline docs/openapi/components/schemas/common/SC237AForm.yml:4]: Two suggestions: 
1. Please prefix with `USDA`
2. Spell out "SC" to make this less esoteric. 

I would suggest naming this `USDASpecialtyCrops237AForm`.

**Review by nissimsan — CHANGES_REQUESTED** [review]

File: docs/openapi/components/schemas/common/SC237AForm.yml (hunk 2 above)
Comment (rhofvendahl) [reply docs/openapi/components/schemas/common/SC237AForm.yml]: @nissimsan I just made this change, could you mark this request as complete so it doesn't block the merge?

Comment (nissimsan) [reply docs/openapi/components/schemas/common/SC237AForm.yml]: Excellent

**Review by nissimsan — APPROVED** [review]

File: docs/openapi/components/schemas/common/SC237AForm.yml (hunk 1 above)
Comment (mkhraisha) [reply docs/openapi/components/schemas/common/SC237AForm.yml]: yes a detailed description instead works.

**Review by mkhraisha — CHANGES_REQUESTED** [review]
Please update the description

**Comment by rhofvendahl** [comment]
@mkhraisha I've updated the description for clarity, should be ready to merge.

**Review by OR13 — APPROVED** [review]

**Pull request merged**